cobalt/browser/cobalt_content_browser_client.cc

```diff
@@ -72,6 +75,14 @@ constexpr base::FilePath::CharType kTransportSecurityPersisterFilename[] =
 constexpr base::FilePath::CharType kTrustTokenFilename[] =
     FILE_PATH_LITERAL("Trust Tokens");
 
+std::string GetApplicationLocaleHelper() {
+#if BUILDFLAG(IS_ANDROID)
+  return std::string(SbSystemGetLocaleId());
+#else
+  return base::i18n::GetConfiguredLocale();
+#endif
```

Comment:
This seems backwards from my naive knowledge, right?
Should be AndroidTV then query base, if not-Android-TV,
i.e. Evergreen then query Starboard?